### PR TITLE
upd(discord): `0.0.49` -> `0.0.50`

### PR DESCRIPTION
**Pull request by ItsLynix**

### CODE REVIEW

**Comment by ItsLynix** [comment]
I would also like to add that this pacscript is faulty in that it doesn't chmod 755 to /usr/share/discord and as such the desktop entry can't launch it, is it okay to do a fix as simple as adding the command to the pacscript command list?

**Comment by ItsLynix** [comment]
> I would also like to add that this pacscript is faulty in that it doesn't chmod 755 to /usr/share/discord and as such the desktop entry can't launch it, is it okay to do a fix as simple as adding the command to the pacscript command list?

Did a little test, adding sudo chmod 755 $(pkgdir)/usr/share/discord to the pacscript works to fix that, but I will wait further comment before making any change

**Comment by Elsie19** [comment]
> > I would also like to add that this pacscript is faulty in that it doesn't chmod 755 to /usr/share/discord and as such the desktop entry can't launch it, is it okay to do a fix as simple as adding the command to the pacscript command list?
> 
> Did a little test, adding sudo chmod 755 $(pkgdir)/usr/share/discord to the pacscript works to fix that, but I will wait further comment before making any change

If it launches when you add it, go for it.

File: packages/discord/discord.pacscript [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
 pkgdesc="Chat for Communities and Friends"
-hash="ad54980244d94e5bab9db51e60914282c3dcb02aa32756c99de4108b2866bf04"
+hash="e955dd54b93b67c34640c89276005177c348b9e524b6421de815d828d0016fee"
 arch=('amd64')
 repology=("project: ${name}")
 
 package() {
   sudo install -dm644 "${pkgdir}/usr/bin"
   sudo install -dm644 "${pkgdir}/usr/share/discord"
+  sudo chmod 755 $(pkgdir)/usr/share/discord
```
Comment (Elsie19) [inline packages/discord/discord.pacscript:15]:
```suggestion
  sudo chmod 755 "$(pkgdir)/usr/share/discord"
```

**Comment by ItsLynix** [comment]
> hi @ItsLynix
> 
> ```
> sudo install -dm644 "${pkgdir}/usr/share/discord"
> sudo chmod 755 "$(pkgdir)/usr/share/discord"
> ```
> 
> Maybe you can just chmod 755 to /usr/share/discord/Discord which is the binary ?
> 
> in the .desktop file you have :
> 
> `Exec=/usr/share/discord/Discord`
> 
> (I'm also not fan about installing 3rd party apps indo usr/share, opt is the repository that normaly must be used for that. I don't understand why Discord choose usr/share as default...)

I believe this doesn't help since it's the actual folder /usr/share/discord that isn't accessible to the user.

**Pull request closed without merging**